db/migrations/000002_insert_sample_data.up.sql

```diff
@@ -0,0 +1,48 @@
+BEGIN;
+
+-- Insert sample data into `users`
+INSERT INTO users (id, name, email, password_hash, created_at, updated_at)
+VALUES
+('11111111-1111-1111-1111-111111111111', 'User One', 'user1@example.com', 'passwordhash1', NOW(), NOW()),
```

Comment:
Do not use such GUIDS - they are valid, but as you may see you reuse the same guid for different entities, that is wrong. I personally use free online GUID generators like https://guidgenerator.com